pkg/config/setup/apm.go

```diff
@@ -152,6 +152,7 @@ func setupAPM(config pkgconfigmodel.Setup) {
 	config.BindEnv("apm_config.install_time", "DD_INSTRUMENTATION_INSTALL_TIME")
 	config.BindEnv("apm_config.obfuscation.credit_cards.enabled", "DD_APM_OBFUSCATION_CREDIT_CARDS_ENABLED")
 	config.BindEnv("apm_config.obfuscation.credit_cards.luhn", "DD_APM_OBFUSCATION_CREDIT_CARDS_LUHN")
+	config.BindEnv("apm_config.obfuscation.credit_cards.skip_keys", "DD_APM_OBFUSCATION_CREDIT_CARDS_SKIP_KEYS")
```

Comment:
other obfuscators under `apm_config` use `keep_values` for the allowlist. Maybe we should keep it consistent with them?